setup/install/providers/aws/aws-ec2.md

```diff
@@ -25,10 +25,13 @@ Use this option to deploy Spinnaker, if you are familar with deployment using [A
 1. Navigate to [Console](https://console.aws.amazon.com/){:target="\_blank"} > CloudFormation and [select](https://docs.aws.amazon.com/awsconsolehelpdocs/latest/gsg/getting-started.html#select-region) your preferred region.
 2. Download [the template](https://d3079gxvs8ayeg.cloudfront.net/templates/managing.yaml) locally to your workstation.
 
-    2.a (Optional). Add additional managed account as shown on line 158 in the SpinnakerAssumeRolePolicy section of the downloaded template file.
-3. Creating the CloudFormation Stack
+    2.a Search for ‘SpinnakerInstanceProfileArn’ and comment out the line.
+    
+    2.b (Optional). Add additional managed account as shown on line 158 in the SpinnakerAssumeRolePolicy section of the downloaded template file.
```

Comment:
```suggestion
    2.b (Optional). Add additional managed account as shown in the `SpinnakerAssumeRolePolicy` section of the downloaded template file.
```

Reply:
@anooptejt we should go ahead and remove it anyway, even though it was already there. I think it should be sufficient for us just to cite the `SpinnakerAssumeRolePolicy` element, and let the user search for that.

Reply:
Sounds, Good!!! I am committing this Suggestion..